src/server/editorServices.ts

```diff
@@ -1373,6 +1387,7 @@ namespace ts.server {
             project.print(/*writeProjectFileNames*/ true);
 
             project.close();
+            removeProjectFromSharedExtendedConfigFilesWatch(project, this.sharedExtendedConfigFilesMap);

```

Comment:
Please have these methods as private methods on the projectServerice instead of in the watchUtilities